src/Events/CategoriesList/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import CategoriesList from "./CategoriesList";
+import { showAllCategories } from "../../actions/category";
+
+class CategoriesListContainer extends Component {
+  componentDidMount() {
+    console.log("this.props @CategoriesList comp", this.props);
```

Comment:
you have an active console log on each component, this is surely making debbuging stuff harder since you get lots of consoles. use them to debug something and then remove them, if you need them you add them again